wbstack/src/Settings/LocalSettings.php

```diff
@@ -22,7 +22,7 @@
 }
 
 // Define some conditions to switch behaviour on
-$wwDomainSaysLocal = $_SERVER['SERVER_NAME'] === 'localhost';
+$wwDomainSaysLocal = preg_match("/(\w\.localhost)/", $_SERVER['SERVER_NAME']) === 1;
```

Comment:
It might be worth hiding all of this localhost stuff behind an environment variable check at some point.
This would avoid such preg_match calls (however small) etc from being called in real production, and would also likely mean that we would never accidently do anything bad.